.github/workflows/build.yaml

```diff
@@ -66,3 +66,45 @@ jobs:
       node_type: "gpu-v100-latest-1"
       run_script: "ci/build_docs.sh"
       sha: ${{ inputs.sha }}
+  wheel-build-cpp:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-24.06
+    with:
+      matrix_filter: group_by([.ARCH, (.CUDA_VER|split(".")|map(tonumber)|.[0])]) | map(max_by(.PY_VER|split(".")|map(tonumber)))
```

Comment:
Out-of-scope for this PR, but this `matrix_filter` is reused in two workflow files in this PR as well as in https://github.com/rapidsai/rmm/pull/1554. I suspect it will be used in most (every?) repo making these changes, so porting it as an option in `rapidsai/shared-workflows` eventually might be good.

Reply:
Agreed, this is in scope for https://github.com/rapidsai/build-planning/issues/43 and https://github.com/rapidsai/build-planning/issues/59.